airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -2109,3 +2128,265 @@ def execute(self, context: Any):
+        :param validation_type: For the set of data to be validated (i.e. compared against expectations), is it already
+            sitting in a BigQuery table or do you want to validate the data returned by a SQL query?  The options are
+            'TABLE' or 'SQL'.
+        :type validation_type: str
+        :param validation_type_input:  The name of the BigQuery table (dataset_name.table_name) if the validation_type
+            is 'TABLE' or the SQL query string if the validation_type is 'SQL'.
+        :type validation_type_input: str
+        :param bigquery_conn_id: Name of the BigQuery connection that contains the connection and credentials
+            info needed to connect to BigQuery.
+        :type bigquery_conn_id: str
+        :param bq_dataset_name:  The name of the BigQuery data set where any temp tables will be created that are needed
+            as part of the GE validation process.
+        :type bq_dataset_name: str
+        :param send_alert_email:  Send an alert email if one or more expectations fail to be met?  Defaults to True.
+        :type send_alert_email: boolean
+        :param include_datadocs_link_in_email:  Include in the alert email a link to the data doc in GCS that shows the
+            validation results?  Defaults to False because there's extra setup needed to serve HTML data docs stored in
+            GCS.  When set to False, only a GCS path to the results are included in the email.
+            Execute the following steps if you want a clickable link for the data doc to be included in the
+            email:
+            1) Set up a GAE app to serve the data docs by following the instructions here:
+            https://docs.greatexpectations.io/en/latest/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_gcs.html
+            2) Create an Airflow variable called 'great_expectations_datadocs_domain' with a value set to the domain
+               google creates in step 1 to serve the data docs (e.g. 'ge-data-docs-dot-my-gcp-project.ue.r.appspot.com').
+               This operator will look for it if include_datadocs_link_in_email=True
+        :type include_datadocs_link_in_email: boolean
+        :param email_to:  Email address to receive any alerts when expectations are not met.
+        :type email_to: str
+        :param fail_if_expectations_not_met: Fail the Airflow task if expectations are not met?  Defaults to True.
+        :type fail_if_expectations_not_met: boolean
+    """
+
+    @apply_defaults
+    def __init__(self, gcp_project, expectations_file_name, gcs_bucket, gcs_expectations_prefix,
+                 gcs_validations_prefix, gcs_datadocs_prefix, validation_type, validation_type_input,
+                 bq_dataset_name, email_to, send_alert_email=True, include_datadocs_link_in_email=False,
+                 fail_if_expectations_not_met=True,
+                 bigquery_conn_id='bigquery_default',
+                 *args, **kwargs):
```

Comment:
Please apply `pre-commit run black` to this file. We started using black formatter for providers package. Also please add type hints and as only keywords are allowed in operators we should remove `*args` and use the following signature `__init__(self, *, gcp_project, ..., **kwargs)` In this way python will know that only keywords are allowed (take a look at other operators).